src/Rule/IpHandler.php

```diff
@@ -113,11 +109,11 @@ private static function validateVersion(
         string $value,
         ValidationContext $context
     ): Result|null {
-        if ($ipVersion === IpHelper::IPV6 && !$rule->isAllowIpv6()) {
+        if ($ipVersion === IpHelper::IPV6 && !$rule->isIpv6Allowed()) {
             return self::getGenericErrorResult($rule->getIpv6NotAllowedMessage(), $context, $value);
         }
 
-        if ($ipVersion === IpHelper::IPV4 && !$rule->isAllowIpv4()) {
+        if ($ipVersion === IpHelper::IPV4 && !$rule->isIpv4Allowed()) {
             return self::getGenericErrorResult($rule->getIpv4NotAllowedMessage(), $context, $value);
         }
 
```

Comment:
The methods are not documented:

- [ ] `getIpParsePattern()` - has description only.
- [ ] `validateValueParts`
- [ ] `validateVersion`
- [ ] `validateCidr`
- [ ] `getGenericErrorResult`